dtfs-central-api/src/services/utilisation-report-data-validator/utilisation-report-cell-validators/generate-total-fees-accrued-exchange-rate-error.ts

```diff
@@ -1,21 +1,24 @@
-const { UTILISATION_REPORT_HEADERS } = require('@ukef/dtfs2-common');
-const { FILE_UPLOAD } = require('../../../../constants/file-upload');
-const { EXCHANGE_RATE_REGEX } = require('../../../../constants/regex');
+import { UTILISATION_REPORT_HEADERS } from '@ukef/dtfs2-common';
+import { FILE_UPLOAD } from '../../../constants/file-upload';
+import { EXCHANGE_RATE_REGEX } from '../../../constants/regex';
+import { UtilisationReportRowValidationErrorGenerator } from './types/validation-error-generator';
 
 /**
  * Validates an object representing a row of csv data to check if the total fees accrued exchange rate is valid.
- * @param {unknown} csvDataRow - object representing a row of csv data.
- * @returns {object | null} - object comprising of error message and error location or null if valid.
+ * @param csvDataRow - object representing a row of csv data.
+ * @returns - object comprising of error message and error location or null if valid.
  */
-const generateTotalFeesAccruedExchangeRateError = (csvDataRow) => {
+export const generateTotalFeesAccruedExchangeRateError: UtilisationReportRowValidationErrorGenerator = (csvDataRow) => {
+  const totalFeesAccruedExchangeRateValue = csvDataRow[UTILISATION_REPORT_HEADERS.TOTAL_FEES_ACCRUED_EXCHANGE_RATE]?.value;
+
   if (
-    !csvDataRow[UTILISATION_REPORT_HEADERS.TOTAL_FEES_ACCRUED_EXCHANGE_RATE]?.value &&
+    !totalFeesAccruedExchangeRateValue &&
     (!csvDataRow[UTILISATION_REPORT_HEADERS.TOTAL_FEES_ACCRUED_CURRENCY]?.value ||
```

Comment:
could these each be const and the or statement be another const.  Will be easier to read